pkg/csi/service/wcp/controller_helper.go

```diff
@@ -100,67 +99,71 @@ func validateWCPControllerUnpublishVolumeRequest(ctx context.Context, req *csi.C
 // validateWCPControllerExpandVolumeRequest is the helper function to validate
 // ExpandVolumeRequest for WCP CSI driver.
 // Function returns error if validation fails otherwise returns nil.
-func validateWCPControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest, manager *common.Manager) error {
+func validateWCPControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest,
+	manager *common.Manager, containerOrchestratorUtility commonco.COCommonInterface) error {
 	log := logger.GetLogger(ctx)
 	if err := common.ValidateControllerExpandVolumeRequest(ctx, req); err != nil {
 		return err
 	}
 
-	var nodes []*cnsvsphere.VirtualMachine
+	if !containerOrchestratorUtility.IsFSSEnabled(ctx, common.OnlineVolumeExtend) {
```

Comment:
Can you explain what have you changed in this function? Are you just enabling validation if online volume expand feature is not enabled?

Reply:
Yes, just the FSS enablement has been added to this code